test/views/ConsigneeDetailsViewSpec.scala

```diff
@@ -47,70 +47,67 @@ class ConsigneeDetailsViewSpec extends ViewSpec with ConsigneeDetailsMessages wi
       getElementByCss(createView(), "title").text() must be(messages(title))
     }
 
-    "display header with hint" in {
+    "display header only" in {
 
       val view = createView()
 
       getElementByCss(view, "legend>h1").text() must be(messages(title))
-      getElementByCss(view, "legend>span").text() must be(messages(hint))
     }
 
     "display empty input with label for EORI" in {
 
       val view = createView()
 
-      getElementByCss(view, "label.form-label>span").text() must be(messages(eori))
-      getElementByCss(view, "label.form-label>span.form-hint").text() must be(messages(eoriHint))
+      getElementByCss(view, "label.form-label>span").text() must be(messages("supplementary.consignee.eori.info"))
+      getElementByCss(view, "label.form-label>span.form-hint").text() must be(messages("supplementary.consignee.eori.hint"))
```

Comment:
The constants `eori` and `eoriHint` are from `CommonMessages`. You can create new constants in `ConsigneeDetailsMessages` now, to avoid using whole paths in tests.

Reply:
@leszeklorek your influence is being felt! :wink: 
The Force is strong with you.